CreateSigningDirectory.cmd

```diff
@@ -25,21 +25,32 @@ call :CreateDirIfNotExist %SigningDirectory%
 call :CreateDirIfNotExist %SigningDirectory%\net461\
 call :CreateDirIfNotExist %SigningDirectory%\netcoreapp2.0
 
-call :CopyFilesForMultitargeting Json.Schema.dll            || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Pointer.dll           || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.ToDotNet.dll   || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.Validation.dll || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.dll            || goto :ExitFailed
```

Comment:
>CopyDllForMultitargeting [](start = 6, length = 24)

Consider: `CopyLibraryToSigningDirectory`. See the reasoning in my comments on `PushSignedBinariesIntoBuild.cmd`.